apps/nextjs-website/react-components/components/Cards/Cards.helpers.tsx

```diff
@@ -0,0 +1,28 @@
+import { Grid, Stack } from '@mui/material';
+import { type ReactNode } from 'react';
+
+export const CardsItemContainer = ({
+  masonry,
+  children,
+}: {
+  masonry: boolean;
+  children: ReactNode[];
```

Comment:
```suggestion
  children: CardsItem[];
```
Would this be ok since we should only be passing CardsItems to this container